src/components/svg/Icons.tsx

```diff
@@ -661,6 +661,90 @@ export const ManyDocumentsIcon = ({ height = "24", width = "24", color = "curren
   );
 };
 
+export const FacebookIcon = ({ height = "20", width = "24", color = "#0171B8" }: IconProps) => {
```

Comment:
Use `"currentColor"` rather than explicitly using a hex code otherwise if we ever want to change the brand colours we will need to manually go through each icon.

The colour can then be set on the container, which should permeate into the icon